nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/WithFrameworkProperties.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  Contributors:
+ *      Kevin Leturc <kleturc@nuxeo.com>
+ */
+
+package org.nuxeo.runtime.test.runner;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Repeatable;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.nuxeo.runtime.api.Framework;
+
+/**
+ * @since 11.1
+ */
+@Inherited
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface WithFrameworkProperties {
```

Comment:
i am wondering if this annotation can be add on `Type` with adding `ElementType.TYPE` i agree if we do it we should manage the restoring process. i think it will be easy if a whole test need to configure the same property for example sending emails... ;)wdyth

Reply:
I think it's not useful right now regarding the complexity to handle `Type` and the usage we may have currently.

I'm in favor to keep the feature as it for now, and improve it later  when such usage will appear, wdyt?

Reply:
yep, i agree 👍